app-of-apps/envs/ocp-prod/clusterwide-entitlements/application.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: clusterwide-entitlements
+spec:
+  destination:
+    name: ocp-prod
+    namespace: open-cluster-management-agent
+  project: mocops
+  source:
+    path: .
+    repoURL: https://github.com/CCI-MOC/ocp-prod-clusterwide-entitlements.git
```

Comment:
We probably want to create a "deploy key" for accessing this repository. That means the public key is associated specifically with the repository and not with your account.

That also means this needs to be an `ssh://` url instead of `https://`.

Reply:
I changed it to git@github.... format to match the URL format in the secret.